howso/engine/trainee.py

```diff
@@ -3833,6 +3854,191 @@ def __exit__(self, exc_type, exc_val, exc_tb) -> None:
             if error.status != 404:
                 raise
 
+    def get_contribution_matrix(
+        self,
+        features: Optional[dict[str, str]] = None,
+        robust: bool = True,
+        targeted: bool = False,
+        normalize: bool = False,
+        normalize_method: str = "relative",
+        absolute: bool = False,
+        fill_diagonal: bool = True,
+        fill_diagonal_value: Union[float, int] = 1,
+    ) -> DataFrame:
+        """
+        Gets the Feature Contribution matrix.
+
+        Parameters
+        ----------
+        features : Optional[Iterable[str]]
+            An iterable of feature names. If features are not provided, then the default trainee
+            features will be used.
+        robust : bool, default True
+            Whether to use robust calcuations.
+        targeted : bool, default False
+            Whether to do a targeted re-analyze before each feature's contribution is calculated.
+        normalize : bool, default False
+            Whether to normalize the matrix row wise. If postive and negative values are present, the normalized values
+            will be between -1 and 1. If only positive values are present, then normalized values will be between 0 and
+            1.
+        normalize_method : str, default 'relative'
```

Comment:
This parameters docstring needs to be updated to include the callable option as well I think